### PR TITLE
Meta

### LINKED ISSUE #110: Meta
My sources of inspiration
* Twitter
* R-pkg-devel that I read more or less regularly
* RStudio community forum, package development category, ends up in the same folder of my inbox
* My own problems

My sources of information
* The same as the sources of inspiration (i.e. other folks' answers)
* The R packages book for context
* Writing R Extensions for all the things. Not reading it all at once, only the piece important for the topic at hand.
* CRAN policies
* github.com/cran (to find examples) and github.com/wch/r-source (to see what actually happens)
* rOpenSci dev guide, sometimes old rOpenSci submission threads
* search engine and other places to find out about packages, package collections, to find relevant packages for a topic

### PR DESCRIPTION
**Pull request by maelle**
Fix #110 

_very_ meta

### CODE REVIEW

**Comment by maelle** [comment]
:wave: @gaborcsardi! Would you have time to have a look at this? Thank you!

**Comment by gaborcsardi** [comment]
Sure, sry for te delay.

**Comment by maelle** [comment]
Np, I'd like to publish it this week @gaborcsardi but it wouldn't be dramatic if it waits a bit more.

**Comment by maelle** [comment]
:wave: @gaborcsardi

**Comment by gaborcsardi** [comment]
I am sorry, I messed this up. :(

**Comment by maelle** [comment]
np, it can wait until next week!

**Comment by maelle** [comment]
:wave: @gaborcsardi

**Comment by maelle** [comment]
:wave: @gaborcsardi

File: content/post/2020-07-15-topic-research/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+)
+
+```
+
+An alternative title for this post would be "the very meta one". :wink:
+In this post we shall present some principles and tips used in preparing posts on this blog, that might be useful for problem solving for R package developers and for blogging here or elsewhere.
+
+## Choosing and finding relevant topics
+
+A topic is relevant if it's related to R package development, and if it's covered extensively elsewhere: unless a blog post can provide more context, presentation of newer tooling, a new perspective, we are not going to duplicate existing content.
```
Comment (gaborcsardi) [inline content/post/2020-07-15-topic-research/index.Rmd:41]: if it's _not_ covered?

**Review by gaborcsardi — APPROVED** [review]
Sorry this took forever. A very meta post indeed. Had a comment.

**Pull request merged**